pyproject.toml

```diff
@@ -122,11 +124,16 @@ matrix-is-tester = {git = "https://github.com/matrix-org/matrix-is-tester", rev
 mypy = ">=0.902"
 mypy-zope = ">=0.3.1"
 parameterized = "==0.8.1"
+# sentry_sdk is required for typechecking.
+sentry_sdk = "*"
```

Comment:
Above you write `sentry-sdk` with a hyphenminus; here an underscore. Should this be a hyphenminus too?

Reply:
Yes, thank you. (The distribution name is [`sentry-sdk`](https://pypi.org/project/sentry-sdk/); the package it provides is `sentry_sdk`.)